service/controller/resource/deployment/template/main.json

```diff
@@ -428,7 +428,7 @@
                       "sourcePortRange":"*",
                       "destinationPortRange":"22",
                       "sourceAddressPrefix":"[parameters('hostClusterCidr')]",
-                      "destinationAddressPrefix":"[parameters('workerSubnetCidr')]",
+                      "destinationAddressPrefix":"[parameters('virtualNetworkCidr')]",
```

Comment:
You changed a .json file but I don't see the pkged.go file. Did you run pkger?

Reply:
Plus it doesn't seem this parameter is defined in the ARM template

Reply:
:scream: 

Totally forgot the whole thing.

Reply:
Done. Tested. W0krs. Reverted `project.go`.